packages/apps/transfer/src/pages/transfer/cross-chain-transfer-tracker/index.tsx

```diff
@@ -1,12 +1,10 @@
 import MainLayout from '@/components/Common/Layout/MainLayout';
 import React, { FC } from 'react';
 
-const CrossChainTransferTracker: FC = () => {
-  return (
-    <MainLayout title="Kadena Coin Transfer">
-      <h3>TO DO: Cross Chain Transfer Tracker</h3>
-    </MainLayout>
-  );
-};
+const CrossChainTransferTracker: FC = () => (
+  <MainLayout title="Kadena Coin Transfer">
+    <h3>TO DO: Cross Chain Transfer Tracker</h3>
+  </MainLayout>
+);
```

Comment:
It seems like we have decided to keep the code as it is after our discussion.

I.e.: when its written like `const x = () => { return "" }` it would **not** be rewritten to `const x = () => ""`

Reply:
ah no, this is coming from previous linter when I enabled that rule, now it's not enabled so it won't effect anything, this is just an old change